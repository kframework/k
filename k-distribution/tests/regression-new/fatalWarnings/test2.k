module TEST2-SYNTAX

endmodule

module TEST2
imports TEST2-SYNTAX

syntax Foo ::= "foo" "(" Foo ")"

rule foo ( A ) => A

endmodule
