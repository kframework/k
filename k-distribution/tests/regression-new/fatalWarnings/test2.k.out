[Error] Compiler: Had 1 parsing errors.
[Error] Inner Parser: Parsing ambiguity. Arbitrarily choosing the first.
1: syntax Bar ::= "foo" [klabel(bar) symbol()]
    bar(.KList)
2: syntax Foo ::= "foo" [klabel(foo) symbol()]
    foo(.KList)
	Source(test2.k)
	Location(11,8,11,11)
